src/components/common/accordion/Accordion.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classnames from 'classnames';
+import { AccordionProps, BodyProps, HeaderProps } from './types';
+
+const Header: React.FC<HeaderProps> = ({ children, ...props }) => {
+  const randomId = Math.random().toString(36);
+  const { className, id = randomId } = props;
+  const classNames = classnames('accordion-header', className);
+
+  return (
+    <React.Fragment>
+      <input type="checkbox" id={id} hidden />
+      <label className={classNames} htmlFor={id}>
+        {children}
+      </label>
```

Comment:
avoid using id for inputs especially in react, wrap input in label

Reply:
Accordion refactored